src/components/TodoComponent/TodoComponent.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  id: number;
+  title: string;
+  completed: boolean;
+  isSelectedTodo: boolean;
+  onTodoSelected: (todo: Todo) => void;
+};
+
+export const TodoComponent: React.FC<Props> = ({
+  todo,
+  id,
```

Comment:
Remove the 'id', 'title', and 'completed' props from the destructuring assignment, as they are not needed. 💡